scripts/generate-topology.py

```diff
@@ -19,17 +20,42 @@ def main():
     p.add_argument('-i', dest='input', required=True)
     p.add_argument('-o', dest='output', required=True)
     p.add_argument('-c', dest='count', type=int, default=1)
-    p.add_argument('--generated-seed', dest='generated_seed', action='store_true', default=False)
+    p.add_argument('--seed', dest='seed_policy', type=str, default="r", \
+        help="Set seed policy. 'r' for random seed, 'g' for generated seed, 'f' for fixed seed")
```

Comment:
ArgParser supports choices, making any further error checking unnecessary. 

`p.add_argument('--seed', dest='seed_policy', choices=['r', 'g', 'f'], default='r',`

The backslash is not needed, and the next line should be indented further (build warning: "PEP 8: E128 continuation line under-indented for visual indent").